### ISSUE TITLE
Prevent app icon showing in recent apps in macOS dock

### ISSUE THREAD

**Issue by houndtt** [will-stone/browserosaurus#166]
On macOS Catalina I'd really like if when browserosaurus launches that the app doesn't show up in my recently used apps in the dock. Ideally it should only show the menubar icon like other apps I use including OneDrive, Symantec Endpoint Protection or Little Snitch.

As an alternative, you can consider having the app launch as one which does not retain in the recent app list. I notice this with apps like Symantec Endpoint Protection launch they show up as a fourth icon in the recent apps list (even if no apps are running). As such, when they quit they don't remain in the recent apps. Maybe something like this can also be considered?

**Comment by will-stone** [comment]
Hi. Thanks for logging this. I turned off the recent app feature as soon as upgraded to macOS Catalina so have not witnessed it. Can you please tell me how I can reproduce it? I turned on recent apps, quit B and restarted it but nothing is showing in the dock for me.

**Comment by will-stone** [comment]
Totally agree, and that's exactly what this line is supposed to do: https://github.com/will-stone/browserosaurus/blob/master/src/main/main.ts#L293

Thanks for the steps, I will try to replicate and see if the Electron community have found a fix.

**Comment by houndtt** [comment]
I don't know much about Electron however I was looking online and noted the following: [How to hide an app's Dock icon](https://www.macworld.com/article/1153731/hidedockicon.html). Based on this article it seems to refer to a property called `LSUIElement` in the application's `Info.plist`.

Comparing with OneDrive, this property exists as follows in its metadata file:
```
	<key>LSUIElement</key>
	<string>NO</string>
```
I tried copying this into the Browserosaurus metadata file but when launched it results in an app crash.

I then checked with Symantec Endpoint Protection and true enough this property was not in its metadata file. What I also noted is that is that for this app, it did not retain its icon in the dock only when launched from the menu bar app. If the app is launched from the Finder icon it behaves normally in the dock.

I then checked back with OneDrive and noted that within the app package bundle there was another app within called OneDrive File Launcher. This has some different code in its metafile:
```
	<key>LSUIElement</key>
	<true/>
```
Copying this code into Browserosaurus also resulted in a crash but it got me thinking, could the way that the apps prevent them from staying in the recent apps in the dock was via some sort of launcher helper app? Both OneDrive and Symantec Endpoint Protection seem to have some mechanism by which another app launched the main app to behave in this manner.

Hope this helps...
